tests/steps/sth_steps.py

```diff
@@ -89,3 +106,97 @@ def check_attr_exists_all_entities(context, attname, att_values):
     values = att_values.split(";")
     for entity in context.entity_list:
         _check_attr_exists(context, entity, attname, values)
+
+
+@step(u'I check entity with "{entity_id}" and "{entity_type}" does not have the attribute "{attname}" ' +
```

Comment:
in steps definitions is highly recommended to include the var type that you are going to use to bring more info about what is going to be replaced in. This line for instance you have it at the last one but not in the two first vars, I would change it to: 
I check entity with entity id "{entity_id}" and entity type "{entity_type}" does not have the attribute "{attname}"